checker/check.ml

```diff
@@ -280,6 +280,7 @@ type library_objects = lib_objects * lib_objects
 
 type library_disk = {
   md_compiled : Safe_typing.compiled_library;
+  md_syntax_objects : library_objects;
```

Comment:
One thing I was wondering: is there a procedure to follow these days, when changing the format of `.vo` files? (magic number,...)